playwright/README.md

```diff
@@ -0,0 +1,4 @@
+# How to use
+* Install dependencies `$  pip install -r requirements.txt`
+* Copy configuration file `vrt.json` into directory.
```

Comment:
could you specify where to get example file?

Reply:
I was hoping we can have a central doc that specifies the structure and how to put it together. We could then link to that in the examples.